test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -0,0 +1,201 @@
+import localize from '../../support/i18n';
+
+// <reference types="Cypress" />
+const selectedRowClassName = 'bce-item-row-selected';
+const selectedRowClassSelector = `.${selectedRowClassName}`;
+const helpers = {
+    load(additionalProps = {}) {
+        cy.visit('/Elements/ContentExplorer', {
+            onBeforeLoad: contentWindow => {
+                contentWindow.PROPS = additionalProps;
+            },
+        });
+    },
+    // cy.contains('Upload') returns the 'canUpload' help text
+    // at the bottom of the page, as opposed to the Upload button.
+    // Use this function when you need cy.contains() to match exact text.
+    getExactRegex(str) {
+        return new RegExp(`^${str}$`);
+    },
+    getRow(rowNum) {
+        return cy.getByTestId('content-explorer').find(`.bce-item-row-${rowNum}`);
+    },
+    checkRowSelections(selectedRow) {
+        if (selectedRow) {
+            cy.getByTestId('content-explorer')
+                .find(`${selectedRowClassSelector}`)
+                .should('have.length', 1)
+                .should('have.class', `bce-item-row-${selectedRow}`);
+        } else {
+            cy.getByTestId('content-explorer').should('not.have.descendants', selectedRowClassSelector);
+        }
+    },
+    selectRow(rowNum) {
+        this.getRow(rowNum)
```

Comment:
I think you can just `return` this chain because the yielded subject should still be the row after the click and should assertion